.github/workflows/images.yaml

```diff
@@ -46,6 +46,7 @@ jobs:
           context: docker
           build-args: |
             platform=${{ matrix.platform }}
+            BUILDKIT_INLINE_CACHE=1
```

Comment:
This is not needed and shouldn't make a difference since we are using buildx to build. Buildkit is used always by buildx

Reply:
Buildx uses the docker cache by default, which leaves lots to be desired. This enables the buildkit cache, which is much better.